internal/pkg/node/mocknode.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package node
```

Comment:
Move to `pkg/overlay/mockoverlay`

Reply:
I think it makes more sense to keep mocks all in one place.